build.sbt

```diff
@@ -19,8 +19,8 @@ libraryDependencies ++= {
   val json4sVersion = "3.5.0"
 
   Seq(
-    "org.apache.kafka" % "connect-api" % kafkaVersion,
-    "org.apache.kafka" % "connect-json" % kafkaVersion,
+    "org.apache.kafka" % "connect-api" % kafkaVersion % "provided",
+    "org.apache.kafka" % "connect-json" % kafkaVersion % "provided",
     "com.microsoft.azure" % "azure-eventhubs" % azureEventHubSDKVersion,
```

Comment:
The connector expects Kafka v0.10 to be installed. I have added a line to the README to say so.